docs/Github-workflow.md

```diff
@@ -0,0 +1,69 @@
+The Performance team uses GitHub to manage all code and related discussions for the Performance Lab plugin. Please follow the workflow below to ensure that all issues are properly tracked.
```

Comment:
Can you add a `#` heading to the top of this doc, and also a link back to the overview, like it's in the other docs files?